### DISCUSSION TITLE
Citing this package?

### DISCUSSION THREAD

**Discussion by stelmo** [brianguenter/FastDifferentiation.jl#99] [Q&A] [answered]
Hi there! We are in the final stages of preparing a paper that uses FastDifferentiation, and we would like to cite it. Do you have any preferences regarding what/how to cite this work?

**Comment by brianguenter** [comment, marked as answer]
Don't know which citation format you are using. Here are a couple

APA:
brianguenter, Lasse Peters, Neven Sajko, Yingbo Ma, Guillaume Dalle, & elaine-ran. (2024). brianguenter/FastDifferentiation.jl: v0.4.1 (v0.4.1). Zenodo. https://doi.org/10.5281/zenodo.13799349

IEEE:
[1]brianguenter, Lasse Peters, Neven Sajko, Yingbo Ma, Guillaume Dalleand elaine-ran, “brianguenter/FastDifferentiation.jl: v0.4.1”. Zenodo, Sep. 19, 2024. doi: 10.5281/zenodo.13799349.

**Reply by stelmo** [reply]
Thank you!
